### PR TITLE
Add new /partners page to website

### LINKED ISSUE #12: [Partners] Add new /partners page to our site
## Before you `start work`

Please read our contribution [guidelines](https://docs.marketprotocol.io/#contributing) and if there is a bounty involved  please also see [here](https://docs.marketprotocol.io/#gitcoin-and-bounties)

If you have ongoing work from other bounties with us where funding has not been released, please do not pick up a new issue.  We would like to involve as many contributors as possible and parallelize the work flow as much as possible.

Please make sure to comment in the issue here immediately after starting work so we know your plans for implementation and a timeline.

Please also note that in order for work to be accepted, all code must be accompanied by test cases as well.

### User Story

As a user I would like to know about all the awesome projects MARKET Protocol is partnered with and also how to become a partner.

### Why Is this Needed?

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*: Provide awareness for our partners, as well as potential new partners

### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Feature
image

### Solution
[comment]: # (Provide a summary of the solution and a task list on what needs to be fixed.)
*Summary*: Using the provided [.sketch file](https://github.com/MARKETProtocol/assets/blob/master/MockUps/MARKET_Protocol-website.sketch) file implement the /partners page

### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)
- [ ] Implement provided design from .sketch in React-Static
- [ ] Must be fully responsive and use existing styled components wherever possible
- [ ] PR must include tests for all code created in order to be accepted.

### Additional Information
[comment]: # (Any other information that would be useful, content, screenshots, etc.)
see the partners page in the [.sketch file](https://github.com/MARKETProtocol/assets/blob/master/MockUps/MARKET_Protocol-website.sketch)

### PR DESCRIPTION
**Pull request by brentoneill**
##### Description

This PR will add a /partners page to the site. Includes updates to make the Social Links their own component w/ test as well as creating a Newletter Subscribe Banner component w/ tests.

##### Checklist

- [x] Linter status: 100% pass
- [x] Changes don't break existing behavior
- [x] Tests coverage hasn't decreased

##### Affected core subsystem(s)

UI, Tests

##### Testing

Tests have been added (stubbed for now) for the Partners page, SocialLinks components, and Newsletter Subscribe Banner component

##### Refers/Fixes

Fixes #12 

**NOTE:** This is WIP pull request to show progress. 

Remaining items:
- Mobile styling
- Wire up Newsletter Subscribe Banner to the correct actions for signup and "Email Us"
- Additional testing for /partners page and Newsletter Subscribe Banner component

### CODE REVIEW

**Comment by pelsasser** [comment]
@brentoneill - thanks for the WIP PR!

Just a heads up we got the .ai file from our designer so that we can use higher res graphics - 

https://github.com/MARKETProtocol/assets/blob/master/Branding/MARKET_Protocol_Branding_Elements-2.ai

All of the assets should be available in that .ai file.

I will take a look at the code / site in a bit.

File: test/components/SocialLinks.test.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+describe('<SocialLinks />', () => {
+  it('renders without crashing', () => {
+    const component = mount(<SocialLinks />);
+    expect(component.find('SocialLinks').length).toEqual(1);
+  });
+
+  it('renders a link to each one of the social providers: telegram, twitter, medium, and github', () => {
+    const component = mount(<SocialLinks />);
+    console.log();
```
Comment (brentoneill) [inline test/components/SocialLinks.test.tsx]: TODO: remove console.log

**Comment by pelsasser** [comment]
looking good!

**Comment by pelsasser** [comment]
Thanks for the hard work @brentoneill please let me know when your ready for a review!

**Comment by brentoneill** [comment]
@pelsasser - Ready for review!

File: src/components/NewsletterSubscribeBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+              }}
+            >
+              <b>Email Us</b>
+              <Icon type="arrow-right" style={{ fontSize: 18 }} />
+            </Button>
+          </BecomeAPartnerWrapper>
+        </StyledCol>
+        <StyledCol lg={24} xl={8}>
+          <SocialLinksWrapper>
+            <SectionTitle>Market Protocol on the web</SectionTitle>
```
Comment (eswarasai) [inline src/components/NewsletterSubscribeBanner.tsx]: @brentoneill -- Shouldn't this be `MARKET`?

**Comment by eswarasai** [comment]
@brentoneill -- I've couple of requests :

- Can you please update the footer to have the link to the `/partners` page and also remove the coming soon text?

screen shot 2018-05-24 at 5 16 15 pm

- For some weird reason, the hero banner image seems pixelated to me along with couple of other logos -- Maker and Virtuse. @pelsasser -- Can you confirm the same on your end?

@pelsasser -- I'd prefer to have `Become a Partner` as a button indicating a call-to-action instead of having a highlighted text. Let me know your thoughts on this design change.

screen shot 2018-05-24 at 5 23 50 pm

**Comment by pelsasser** [comment]
@brentoneill - thanks! I think overall this looks very good!

I need to update the copy around "Join our mission" but will do that once we merge. 

@eswarasai - I dont have any pixelation issues 

Here are the small issues I do see - 

1. Main "Become a Partner" link doesn't work on mobile resolutions
2. At some widths the text isn't overlayed on top of the hero graphic and at others it is, I like it overlayed!
image
vs
image
3. After seeing this built I think we have alot of redundancy in the footer / CTA sections
image
not sure how it would look if we removed the "Become a Partner" in the CTA section but seems like we should get rid of it. Thoughts on that? perhaps we have a link instead sending people to the team page, or our medium site.  "Learn more about our Team" or "Read more about MARKET Protocol"

**Comment by brentoneill** [comment]
@eswarasai  - I noticed the pixelation issues with the Maker and Virtuse logo as well - do we have a higher res version we can use? I pulled those right out of the sketch file.

@pelsasser  - I meant to have the text overlayed on that image - I liked the way it looked as well. I may just need to adjust the graphic positioning at larger mobile screen sizes.

Do we want to make the "Become a Partner" button look like the button in the CTA section ("Email us ->")

Noted all the other requested changes - will hop on those a bit later today.!

**Comment by pelsasser** [comment]
@brentoneill - looking good!

1. Can we match the button to this mock (1st screenshot)?
image
(current version here for easy comparison)
image
1. There are some widths(~1120) where the CTA gets a bit wonky - 
image  
1. will see what I can do regarding higher res images
1. Still some issues with responsiveness 
image
1. We lost the black header at some widths as well.  
Header
image
No header 
image
1. disregard the comment about the link not working on mobile, seems like it works just fine ;)

**Comment by nitinrgupta** [comment]
@pelsasser I think we can remove the "MARKET Protocol on the web" part from the below section. I had moved that section to the footer which looked like a right placement for me. This would solve the responsive issues as well. What do you think?
40544197-a4561f08-5fe4-11e8-8cc1-6a9e18d81352

**Comment by pelsasser** [comment]
Sounds good to me!

**Comment by brentoneill** [comment]
@pelsasser - Just pushed some fixes for that and remove the "MARKETProtocol" on the web section from the bottom CTA area

**Comment by pelsasser** [comment]
still some responsiveness issues 

image

and
image
vs

image

**Comment by pelsasser** [comment]
p.s sorry about the conflict, modified some alt tags, I can resolve when we merge if need be.

**Pull request merged**